modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -543,6 +565,96 @@ public final void setTabSize(int spaces) {
         tabSizeProperty().set(spaces);
     }
 
+    /**
+     * {@code TabStopPolicy} determines the tab stop positions within this {@code TextFlow}.
```

Comment:
I think you can remove `{@code TabStopPolicy}` since that's what this property is. Start with "Determines the ..."